crates/rust-analyzer/src/main_loop/handlers.rs

```diff
@@ -835,6 +835,7 @@ pub fn handle_code_lens(
     for runnable in world.analysis().runnables(file_id)? {
         let title = match &runnable.kind {
             RunnableKind::Test { .. } | RunnableKind::TestMod { .. } => "▶️\u{fe0e}Run Test",
+            RunnableKind::DocTest { .. } => "▶️\u{fe0e}Run Doctest",
```

Comment:
When a doc comment has multiple tests this should say `Run doctests`.

Reply:
I think its fine :)

Reply:
If you change your mind I already implemented the version with trailing `s` when multiple doc tests. Let me know if you're interested :)